.docs/user-guide/docker-plugins.md

```diff
@@ -408,7 +457,7 @@ plug-in:
 
 Environment Variable | Description | Default | Required
 ---------------------|-------------|---------|---------
-`CINDER_AUTHURL` | The keystone authentication API |  | true
+`CINDER_AUTHURL` | The keystone authentication API |  | ✓
```

Comment:
Hi @codenrhoden,

Not to nitpick, but should this be the same commit as Azure UMD? I don't mind it being the same PR, but maybe rebase it into a separate commit specific to Cinder?

Reply:
will do.

Reply:
This is now on a separate commit.